src/commentGenerator.js

```diff
@@ -0,0 +1,31 @@
+import OpenAI from 'openai'
+console.log('OPENAI_API_KEY:', process.env.OPENAI_API_KEY)
+
+const openai = new OpenAI({
+  apiKey: process.env.OPENAI_API_KEY,
+})
+
+export async function generateComment(postBody) {
+  try {
+    const response = await openai.chat.completions.create({
+      model: 'gpt-4o-mini',
+      messages: [
+        {
+          role: 'system',
+          content: 'You are a witty, funny, and slightly sarcastic bot.',
+        },
+        {
+          role: 'user',
```

Comment:
Prompt can be enhanced a little bit. Try to provide it context like this. 
"You are a SinkedinOfficalBot. Sinkedin is linkedin alternative for flaunting failures and job rejections and embarrasments. Your task is to make fun/roast the content user has posted by creating comment. Not too long, not too short". 

Something like this.